Content/Items/Gear/Weapons/Sword/MagicWoodenSword.cs

```diff
@@ -0,0 +1,41 @@
+using PathOfTerraria.Common.Enums;
+using PathOfTerraria.Core.Items;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Items.Gear.Weapons.Sword;
+
+internal class MagicWoodenSword : Sword
```

Comment:
This is not correct, we don't need another.

We have all the PoT items with extra things on them. 
So you'd set the [Rarity](https://github.com/Path-of-Terraria/PathOfTerraria/blob/main/Common/Enums/ItemRarity.cs#L3) of the item

It should jsut be as simple as applying that, as all items can have a rarity in this.

When an item is spawned with a rarity it "[rolls](https://github.com/Path-of-Terraria/PathOfTerraria/blob/main/Common/ItemSpawner.cs#L105)" 

Generating affixes based on the rarity.

Magic is given 2 affixes, rare is given 4 affixes.

So you'd just apply rarity and go from there I believe.